src/state-manager/TransactionQueue.ts

```diff
@@ -6754,9 +6754,10 @@ class TransactionQueue {
                     if (configContext.stateManager.attachDataToReceipt === false) {
                       if (configContext.p2p.useFactCorrespondingTell) {
                         this.factTellCorrespondingNodesFinalData(queueEntry)
-                      } else {
-                        this.tellCorrespondingNodesFinalData(queueEntry)
-                      }
+                      } 
+                      // else {
```

Comment:
why is this commented?

Reply:
It's deprecated. We use FACT algorithm and the older function is commented out.